sdk/batch/arm-batch/tsconfig.json

```diff
@@ -14,6 +14,6 @@
     "outDir": "./esm",
```

Comment:
You've moved arm-batchai/tsconfig to arm-batch/tsconfig. Two different packages.

Reply:
She moved both tsconfig files as part of this PR one for batch and one for batchai they look correct to me.

Reply:
I see. The diff is very weird.